scripts/tag-versions.sh

```diff
@@ -20,11 +20,12 @@ echo "Application version is $app_version, short version is $short_version."
 dabl_meta=$2
 daml_yaml=$3
 daml_yaml_triggers=$4
-daml_yaml_it=$5
```

Comment:
@alexmatson-da I wasn't sure what this was for - it wasn't used in the rest of the file and had no corresponding argument in the `Makefile`, so I removed it.

Reply:
yep this was for the `integrationTest` directory that had gotten removed a little while ago